.github/workflows/ci.yml

```diff
@@ -91,10 +91,6 @@ jobs:
           - locked
 
         include:
```

Comment:
Why is the lowest testing removed? It is okay to bump minor versions up if necessary to fix "lowest"